packages/ui-components/src/lib/components/deployment/DeploymentSteps.svelte

```diff
@@ -252,7 +252,9 @@
 					{/if}
 
 					{#if allDepositFields.length > 0 && showAdvancedOptions}
```

Comment:
if we have the each do we need the `length > 0` check? if the array is empty the each will render nothing anyway